Standard Gaming Platform/mousesystem.cpp

```diff
@@ -1361,32 +1361,32 @@ void SetRegionFastHelpText( MOUSE_REGION *region, const STR16 szText )
 	//region->FastHelpTimer = gsFastHelpDelay;
 }
 
-INT16 GetNumberOfLinesInHeight( const STR16 pStringA )
-{
-	STR16 pToken;
-	INT16 sCounter = 0;
-	CHAR16 pString[ 4096 ];
-
-	wcscpy( pString, pStringA );
+bool isScalingEnabled() {
+	return fTooltipScaleFactor > 1;
+}
 
-	// tokenize
-	pToken = wcstok( pString, L"\n" );
+UINT16 GetScaledFontHeight()
+{
+	return isScalingEnabled()
+		? GetWinFontHeight(TOOLTIP_IFONT)
+		: GetFontHeight(FONT10ARIAL);

```

Comment:
`GetFontHeight` already returns `GetWinFontHeight` if `iUseWinFonts == true`. might be useful to simplify this further.